packages/buttons/package.json

```diff
@@ -9,23 +9,29 @@
     "url": "https://github.com/zendeskgarden/react-components/issues"
   },
   "version": "2.0.1",
-  "main": "./index.js",
+  "main": "./dist/index.js",
+  "files": [
+    "dist"
+  ],
   "scripts": {
     "build": "../../utils/scripts/build.sh",
     "build:demo": "../../utils/scripts/build-demo.sh",
     "start": "../../utils/scripts/start.sh"
   },
   "dependencies": {
-    "@zendeskgarden/css-buttons": "^6.0.0",
     "@zendeskgarden/react-selection": "^3.0.1",
+    "classnames": "^2.2.5"
+  },
+  "peerDependencies": {
     "@zendeskgarden/react-theming": "^1.0.0",
-    "classnames": "^2.2.5",
     "prop-types": "^15.6.1",
+    "react": "^0.14.0 || ^15.0.0 || ^16.0.0",
+    "react-dom": "^0.14.0 || ^15.0.0 || ^16.0.0",
     "styled-components": "^3.2.6"
   },
-  "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0 || ^16.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0 || ^16.0.0"
+  "devDependencies": {
+    "@zendeskgarden/css-buttons": "^6.0.0",
```

Comment:
So is the idea that due to the proposed change this needs to be explicitly included in your app css now hence it becoming a devDep?

Reply:
The proposed change would remove `@zendeskgarden/css-buttons` from any dependency chain from the consumer.

They now only worry about importing the `react` styling (if there is any).

For buttons it would be:
* `import '@zendeskgarden/react-buttons/styles.css'`
  * This is a transformed version of `@zendeskgarden/css-buttons` with the CSS-modules already transformed for usage with the `react-buttons`
* `import { Button } from '@zendeskgarden/react-buttons'
  * Just like normal

The only reason it's a `devDependency` is for styleguidist and and the webpack build to create the `styles.css` file.